vibe_code_apps_22/bacnet/weather_bacnet_server.py

```
@@ -0,0 +1,244 @@
+            await asyncio.sleep(args.poll)
+            try:
+                sample = fetch_oa(api_key, lat, lon, units)
+                updated = local_updated_stamp()
+                await publish(server, sample, city, updated)
+                log(
+                    f"updated {updated}  {city}  "
+                    f"{sample['temp']}{unit_sym}  "
+                    f"{sample['humidity']}%  "
+                    f"dew {sample['dew_point']}{unit_sym}"
+                )
+            except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)